ros2_control_demo_bringup/config/multi_controller_manager_joint_trajectory_publisher.yaml

```diff
@@ -0,0 +1,42 @@
+/rrbot_1/publisher_joint_trajectory_controller:
+  ros__parameters:
+
+    controller_name: "rrbot_1/position_trajectory_controller"
+    wait_sec_between_publish: 6
+
+    goal_names: ["pos1", "pos2", "pos3", "pos4"]
+    pos1: [0.785, 0.785]
+    pos2: [0.0, 0.0]
+    pos3: [-0.785, -0.785]
+    pos4: [0.0, 0.0]
+
+    joints:
+      - rrbot_1_joint1
```

Comment:
Random question: why does this file include the joints but `ros2_control_demo_bringup/config/multi_controller_manager_forward_position_publisher.yaml` doesn't?

Reply:
Because different type of publisher nodes are used and therefore different type of messages are published. Commands for `JointTrajectoryController` need to have joint names.

This file configures [this](https://github.com/ros-controls/ros2_controllers/blob/master/ros2_controllers_test_nodes/ros2_controllers_test_nodes/publisher_joint_trajectory_controller.py) publisher node.

The above one configured [this](https://github.com/ros-controls/ros2_controllers/blob/master/ros2_controllers_test_nodes/ros2_controllers_test_nodes/publisher_forward_position_controller.py) node.